### PR TITLE
Make usage of abbreviations more consistent

### PR DESCRIPTION
**Pull request by misappi**
* Consolidated abbreviations and made them more consistent
* Introduced "code hosting platform" instead of mentions of a specific hosting company (where it was applicable)
* Consolidated references

### CODE REVIEW

**Review by OliverFendt — COMMENTED** [review]
@misappi great work, looks good to me. Can you please add AI to the list of abbreviations?

* AI = Artificial Intelligence

**Comment by misappi** [comment]
@OliverFendt: Done.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Outbound OSS - Guide
+# Outbound Open Source - Guide
```
Comment (jlprat) [inline README.md:1]: OSS is Open Source Software

**Comment by cornelius** [comment]
Looks good to me.

File: content/02-contributions-to-existing-projects.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# How to contribute to OSS projects
+# How to contribute to open source projects
```
Comment (jlprat) [inline content/02-contributions-to-existing-projects.md:1]: OSS = open source software
Maybe capitalize the O S and S as it is on the first occurrence in the README

File: content/02-contributions-to-existing-projects.md [hunk 3]
```diff
@@ -62,18 +62,18 @@ Finally, public code is like the business card of a company as well as of the de
 
 #### Outbound CLA
```
Comment (jlprat) [inline content/02-contributions-to-existing-projects.md:63]: Should we change CLA with Contributor License Agreement?

Comment (jlprat) [reply content/02-contributions-to-existing-projects.md:63]: This is fine to use

**Comment by misappi** [comment]
@jlprat: Should be OK now. Please approve. Thanks!

File: README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Outbound OSS - Guide
```
Comment (jlprat) [inline README.md:1]: Do you want to add Sofware in here as well?+

Comment (misappi) [reply README.md:1]: That's already changed to "Outbound Open Source - Guide"

Comment (misappi) [reply README.md:1]: And no, I think that "software" is not required here.

**Review by jlprat — APPROVED** [review]
LGTM

**Pull request merged**